service-catalog/generic-service.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    velocity.tech.v1/id: generic-service-var-service-id
+  name: generic-service-var-service-id
+spec:
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      name: generic-service-var-service-id
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        name: generic-service-var-service-id
+    spec:
+      containers:
+        - image: busybox:1.28
+          command:
+            - sleep
+            - "3600"
```

Comment:
change to `infinity`
make sure the container is running 🙏

Reply:
`sleep infinity` doesn't work :(

Reply:
try a newer version of busybox then
https://hub.docker.com/_/busybox
https://stackoverflow.com/questions/63527494/why-busybox-container-is-in-completed-state-instead-of-running-state
I want it to stay alive forever if possible